aip/general/0162.md

```diff
@@ -231,178 +200,80 @@ there are two potential variants:
 - Child resources where each child resource is a child of _a single revision
   of_ the parent resource.
 
-If a child resource is a child of a single revision, the child resource's name
-**must** always explicitly include the parent's revision ID:
-
-    publishers/123/books/les-miserables@c7cfa2a8/pages/42
-
-In `List` requests for such resources, the service **should** default to the
-latest revision of the parent if the user does not specify one, but **must**
-explicitly include the parent's revision ID in the `name` field of resources in
-the response.
-
-If necessary, APIs **may** explicitly support listing child resources across
-parent revisions by accepting the `@-` syntax. For example:
-
-    GET /v1/publishers/123/books/les-miserables@-/pages
-
 APIs **should not** include multiple levels of resources with revisions, as
 this quickly becomes difficult to reason about.
 
-### Committing revisions
+### Standard methods
 
-Depending on the resource, different APIs may have different strategies for
-when to commit a new revision, such as:
+If a revision implements standard methods, they **must** implement the standard
```

Comment:
Is this intended to mean "any standard methods provided must implement the corresponding AIPs"? It's not intended to mean "if you implement some of these you must implement all of them" or anything like that? Happy to try to come up with a clarifying rewording if the former is the intent.

Reply:
the intent was the former. Great wording! added.